stack/rest/src/test/java/org/apache/usergrid/rest/management/RegistrationIT.java

```diff
@@ -17,212 +17,238 @@
-                        .get( String.class );
-                fail( "request for disabled user should fail" );
+                this.management().orgs().organization(organization.getName()).users().user(username)
+                    .getResource(true, getAdminToken())
+                    .get(String.class);
+                fail("request for disabled admin should fail");
+            } catch (UniformInterfaceException uie) {
+                assertEquals(401, uie.getResponse().getStatus());
             }
-            catch ( UniformInterfaceException uie ) {
-                ClientResponse.Status status = uie.getResponse().getClientResponseStatus();
-                JsonNode body = mapper.readTree( uie.getResponse().getEntity( String.class ));
-                assertEquals( "user disabled", body.findPath( "error_description" ).textValue() );
+            setTestProperty(AccountCreationProps.PROPERTIES_SYSADMIN_LOGIN_ALLOWED, "true");
+            Token objToken = management()
+                .token()
+                .post(new Token(username, password));
+            try {
+                clientSetup
+                    .getRestClient()
+                    .org(orgname)
+                    .app(appname)
+                    .users()
+                    .user("testuser")
+                    .revoketokens()
+                    .withToken(objToken)
+                    .put();
+            } catch (UniformInterfaceException uie0) {
+                fail("Failed to revoke user tokens");
             }
-
-            setup.getMgmtSvc().deactivateUser( setup.getEmf().getManagementAppId(), owner_uuid );
             try {
-                resource().path( "/management/token" ).queryParam( "grant_type", "password" )
-                        .queryParam( "username", username ).queryParam( "password", password)
-                        .accept( MediaType.APPLICATION_JSON ).type( MediaType.APPLICATION_JSON_TYPE )
-                        .get( String.class );
-                fail( "request for deactivated user should fail" );
+                clientSetup
+                    .getRestClient()
+                    .org(orgname)
+                    .app(appname)
+                    .users()
+                    .user("testuser")
+                    .withToken(objToken)
+                    .put(appUser.chainPut("activated", "false").chainPut("deactivated", System.currentTimeMillis()));
+                refreshIndex();
+            } catch (UniformInterfaceException uie0) {
+                fail("Failed to deactivate the user");
             }
-            catch ( UniformInterfaceException uie ) {
-                ClientResponse.Status status = uie.getResponse().getClientResponseStatus();
-                JsonNode body = mapper.readTree( uie.getResponse().getEntity( String.class ));
-                assertEquals( "user not activated", body.findPath( "error_description" ).textValue() );
```

Comment:
We still want to read the error_description to make sure we are getting the correct text back. It'll make the refactor of the rest tier errors easier to verify.